index.html

```diff
@@ -1,11 +1,22 @@
 <!DOCTYPE html>
+<!--
+ *  Copyright (c) 2014 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+-->
 <html>
 <head>
 
-  <meta name="keywords" content="WebRTC, HTML5, JavaScript" />
-  <meta name="description" content="Client-side WebRTC code samples." />
-  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
+  <meta charset="utf-8">
+  <meta name="description" content="Client-side WebRTC code samples">
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1, maximum-scale=1">
+  <meta itemprop="description" content="Client-side WebRTC code samples">
+  <meta itemprop="image" content="samples/web/content/apprtc/images/webrtc-icon-192x192.png">
```

Comment:
I think we want to have our own copy of this image, especially if we are going to be moving out AppRTC to its own repo.